src/viam/sdk/rpc/server.cpp

```diff
@@ -1,39 +1,66 @@
 #include <viam/sdk/rpc/server.hpp>
 
-#include <grpcpp/ext/proto_server_reflection_plugin.h>
+#include <boost/log/trivial.hpp>
 #include <grpcpp/security/server_credentials.h>
 
+#include <viam/sdk/registry/registry.hpp>
+
 namespace viam {
 namespace sdk {
 
-Server::Server() : builder_(std::make_unique<grpc::ServerBuilder>()) {}
+Server::Server() : builder_(std::make_unique<grpc::ServerBuilder>()) {
+    auto new_manager = std::make_shared<ResourceManager>();
+    for (const auto& rr : Registry::registered_resources()) {
```

Comment:
Register all known resources when creating the server, to ensure proper gRPC reflection.